configuration/src/main/kotlin/com/malinskiy/marathon/config/serialization/ConfigurationFactory.kt

```diff
@@ -106,10 +107,13 @@ class ConfigurationFactory(
                 VendorConfiguration.StubVendorConfiguration -> configuration.vendorConfiguration
                 is VendorConfiguration.EmptyVendorConfiguration -> throw ConfigurationException("No vendor configuration specified")
             }
-
             return configuration.copy(
                 vendorConfiguration = vendorConfiguration,
-                analyticsTracking = analyticsTracking ?: configuration.analyticsTracking
+                // Default value for analyticsTracking / bugsnagReporting in both CLI and marathon / gradle config is true.
+                // If analyticsTracking / bugsnagReporting is set to false in either CLI or marathon / gradle config, it will be disabled.
+                // Note that it's not possible to set the CLI value when running marathon via gradle.
+                analyticsTracking = !(analyticsTracking == false || configuration.analyticsTracking == false),
```

Comment:
might be easier to rewrite with De Morgan's law into 
`analyticsTracking == true && configuration.analyticsTracking == true`, same below

Reply:
Actually, I don't think it's quite the same because `null` value is allowed for `analyticsTracking`. If `analyticsTracking` is null and `configuration.analyticsTracking` is true, we want it to be enabled (which is what the code currently does). We want to disable it only if one *explicitly* sets it to false for `analyticsTracking` or `configuration.analyticsTracking`.